src/en/index.html

```diff
@@ -337,12 +337,11 @@ <h1>Pikachu Volleyball - Super AI Edition</h1>
 					<p class="reverse-engineered-by">
 						Based on the <a href="https://gorisanson.github.io/pikachu-volleyball/en/" target="_blank">web
 							version</a>
-						developed by <a href="https://gorisanson.github.io/" target="_blank">Lee,
-							Kyutae</a>
+						developed by <a href="https://gorisanson.github.io/" target="_blank">Kyutae Lee</a>
 					</p>
 					<hr style="width: 100%;">
 					<p class="reverse-engineered-by">
-						This version is modified by <span class="no-wrap"><a href="https://blog.duckll.tw/"
+						This Super AI Edition is developed by <span class="no-wrap"><a href="https://blog.duckll.tw/"
```

Comment:
I think "modified" is somewhat weaker expression compared to what you have done. So I change it to "developed". Feel free to let me know your opinion about it :)